mock-fs type doesn't mirror node fs

in the types-2.0 branch:
```
$ cat test/repro.ts
import * as mockFs from 'mock-fs';
mockFs.fs({}).existsSync('');
$ tsc -v
Version 2.0.10
$ tsc
error TS2339: Property 'existsSync' does not exist on type '(config?: Config | undefined, options?: Options | undefined) => typeof fs'.
```

I don't see why this isn't caught by the test. However, there's a simple fix that I'll send a PR for.

More info if it helps:
```
$ cat node_modules/\@types/mock-fs/package.json
{
    "name": "@types/mock-fs",
    "version": "3.6.29",
    "description": "TypeScript definitions for mock-fs 3.6.0",
    "license": "MIT",
    "author": "Wim Looman <https://github.com/Nemo157>, Qubo <https://github.com/tkqubo>",
    "main": "",
    "repository": {
        "type": "git",
        "url": "https://www.github.com/DefinitelyTyped/DefinitelyTyped.git"
    },
    "scripts": {},
    "dependencies": {
        "@types/node": "*"
    },
    "typings": "index.d.ts",
    "typesPublisherContentHash": "7623470e89c244930b343762d7c44b4a8019ffcd5d559f384d4b8625c7c94f23"
}

$ cat node_modules/\@types/mock-fs/types-metadata.json
{
    "authors": "Wim Looman <https://github.com/Nemo157>, Qubo <https://github.com/tkqubo>",
    "definitionFilename": "index.d.ts",
    "libraryDependencies": [
        "node"
    ],
    "moduleDependencies": [
        "fs"
    ],
    "libraryMajorVersion": "3",
    "libraryMinorVersion": "6",
    "libraryName": "mock-fs 3.6.0",
    "typingsPackageName": "mock-fs",
    "projectName": "https://github.com/tschaub/mock-fs",
    "sourceRepoURL": "https://www.github.com/DefinitelyTyped/DefinitelyTyped",
    "sourceBranch": "types-2.0",
    "kind": "ProperModule",
    "globals": [
        "mock"
    ],
    "declaredModules": [
        "mock-fs"
    ],
    "files": [
        "index.d.ts"
    ],
    "hasPackageJson": false,
    "contentHash": "7623470e89c244930b343762d7c44b4a8019ffcd5d559f384d4b8625c7c94f23"
}
```

--- a/mock-fs/index.d.ts
+++ b/mock-fs/index.d.ts
@@ -6,7 +6,7 @@
 /// <reference types="node" />
 
 
-import fs = require("fs");
+import nodefs = require("fs");
 
 declare function mock(config?: mock.Config, options?: mock.Options): void;
 
@@ -17,7 +17,7 @@ declare namespace mock {
 
     function restore(): void;
 
-    function fs(config?: Config, options?: Options): typeof fs;
+    function fs(config?: Config, options?: Options): typeof nodefs;
 
     interface Config {
         [path: string]: string | Buffer | File | Directory | Symlink | Config;

